lib/datadog/lambda.rb

```diff
@@ -54,28 +58,25 @@ def self.configure_apm
     def self.wrap(event, context, &block)
       Datadog::Utils.update_log_level
       @listener ||= initialize_listener
-      @listener.on_start(event: event)
       record_enhanced('invocations', context)
       begin
         cold = @is_cold_start
-        res = @listener.on_wrap(request_context: context, cold_start: cold) do
-          block.call
-        end
+        @listener&.on_start(event: event, request_context: context, cold_start: cold)
```

Comment:
is the &. https://stackoverflow.com/questions/36812647/what-does-ampersand-dot-mean-in-ruby if so is there an scenario we may expect the function to be null?

Reply:
Correct

Reply:
The scenario is where Tracing is not enabled.